### DISCUSSION TITLE
posts/refactoring-ahead-atomic-design-and-custom-hooks

### DISCUSSION THREAD

**Discussion by giscus** [poodlepoodle/poodle-blog-comments-giscus#2] [Comments]
# posts/refactoring-ahead-atomic-design-and-custom-hooks

새로운 기술이 파도처럼 몰려와도 지워지지 않을 개발자국을 남깁니다.

https://www.poodlepoodle.me/posts/refactoring-ahead-atomic-design-and-custom-hooks

**Comment by snacknam** [comment]
정리를 잘 해주셨네요🙂 잘 보고 갑니다~
